### ISSUE TITLE
Minimize output of git-find-ignored

### ISSUE THREAD

**Issue by jmalloc** [make-files/issues#14]
If a file is ignored because the directory it's in is ignored, just print the directory name, not the untracked files within it.

**Comment by ezzatron** [comment]
I'm seeing very verbose output for `make clean` with `pkg-php`. Is that caused by this?

**Comment by jmalloc** [comment]
Yep, I have a snippet somewhere to fix this. I’ll take a look tomorrow.

**Comment by jmalloc** [comment]
For future reference:

```bash
#!/bin/bash
 
pathlist='/foo/
/foo/bar/
/foo/bar/file.txt
/shazam/blammo
/shazam/blammo/file1.txt
/shazam/blammo/file2.txt'
 
i=2
 
while [ $i -lt 100 ]; do
  path=`echo "$pathlist" | cut -f1-$i -d/ | uniq -d`
  if [ -z "$path" ]; then
     echo $prev_path
     break
  fi
  prev_path=$path
  i=`expr $i + 1`
done
```

**Comment by jmalloc** [comment]
There's possibly a better way to solve that, by sorting first and implementing a more obvious `is_parent_of` type function.

**Comment by ezzatron** [comment]
Or, as discussed by voice, we can change `make clean` to use something like:

    git clean -dX --force --exclude !/node_modules/

Where `!/node_modules/` is some kind of list derived from a variable.

**Comment by ezzatron** [comment]
Another issue discussed is the fact that the above approach will clean up things that are "globally" ignored, like IDE-specific project files. We can probably get around this with a command-line option to set the `core.excludesfile` option to empty, i.e.:

    git -c core.excludesfile= clean -dX --force --exclude !/node_modules/

**Comment by ezzatron** [comment]
@jmalloc Should this issue be closed now that make-files/lib-core#9 is merged?

**Comment by jmalloc** [comment]
Yeah, I think it can be closed.

It probably still makes sense to minimise the output of `git-find-ignored` but I'm not sure how that would work given the fact that the script accepts filenames/wildcards are arguments.
